devimint/src/federation.rs

```diff
@@ -32,17 +32,19 @@ use super::vars::utf8;
 use crate::util::{poll, FedimintdCmd};
 use crate::{poll_eq, vars};
 
+#[derive(Clone)]
 pub struct Federation {
```

Comment:
Does it make sense to clone this? Isn't it more like a ... resource?

Reply:
all daemon structs are clone in devimint

Reply:
We don't do things like "kill on `Drop`" etc. on them?

Reply:
that happens in ProcessHandle, which is Arc<ProcessHandleInner>